software/drivers/sd8887/src/wlan/mapp/uaputl/uaputl.h

```diff
@@ -802,6 +802,7 @@ typedef enum {
 	GWK_CIPHER,
 	RESTRICT_CLIENT_MODE,
 	AKM_SUITE,
+	WACPMODE
```

Comment:
@majorz what does WACP stand for?

Reply:
@alexgg this I believe is compatibility feature with Apple CarPlay. That commit in particular came upstream. I only did the ones that start with "Port".

Reply:
I squashed now the commits that came upstream. Now it is cleaner.

Reply:
It is cleaner @majorz, thanks